docker-compose.yml

```diff
@@ -319,6 +334,51 @@ services:
       - ADMIN_PW=$ADMIN_PW
       - API_URL=https://openhim-core:8080
   
+  ## openSHR
+  openshr:
+    container_name: oshr
+    image: ghcr.io/i-tech-uw/oshr:pr-3
+    env_file:  
+      - configs/openshr/.env 
+    # tty: true
+    # stdin_open: true
+    # entrypoint: bash    
+    ports:
+      - 8090:8080
+
+  ## openshr-db
+  openshr-db:
+    container_name: openshr-db
+    image: mysql:5.7
+    command: --sql_mode="NO_ZERO_IN_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"
+    env_file:
+      - configs/openshr/.env
+    volumes:
+      - /var/lib/mysql
+  
+  ## OpenXDS
+  openxds:
+    image: ghcr.io/i-tech-uw/openxds:pr-2
+    container_name: openxds-demo
+    ports:
+      - "8010:8010"
+    command: dockerize -wait tcp://postgresql-openxds:5432 -timeout 100s java -jar /opt/openxds/openxds-1.1.2.jar 
```

Comment:
hi @pmanko  so basing on https://pastebin.com/We0uyaqt ,it seems the openxds.proerties file is not loaded,and we might need to pass it around here